packages/flutter_tools/lib/src/macos/migrations/remove_macos_framework_link_and_embedding_migration.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file is almost the same as https://github.com/flutter/flutter/blob/e1598c96c072a6323e8dbf2a1724a896533a6c27/packages/flutter_tools/lib/src/ios/migrations/remove_framework_link_and_embedding_migration.dart with different identifiers, doesn't do anything with add-to-app, and leaves off any Xcode version checks.